cpp-package/src/board_shim.cpp

```diff
@@ -435,3 +435,14 @@ int *BoardShim::get_temperature_channels (int board_id, int *len)
     }
     return temperature_channels;
 }
+
+int *BoardShim::get_resistance_channels (int board_id, int *len)
+{
+    int *resistance_channels = new int[MAX_CHANNELS];
```

Comment:
I didn't catch this easrlyer butr what are you using "new" to initialize arrays of a constant size?

Reply:
I guess because you are returning the pointer

Reply:
exactly you can not return pointer to local variable